src/topics/topicSelectors.js

```diff
@@ -66,8 +66,8 @@ export const getTopicsForStream: Selector<?(TopicExtended[]), number> = createSe
   },
 );
 
-export const getLastMessageTopic = (narrow: Narrow): Selector<string> =>
-  createSelector(
-    state => getShownMessagesForNarrow(state, narrow),
-    messages => (messages.length === 0 ? '' : messages[messages.length - 1].subject),
-  );
+export const getLastMessageTopic: Selector<string, Narrow> = createSelector(
+  (state, narrow) => narrow,
+  (state, narrow) => getShownMessagesForNarrow(state, narrow),
+  (narrow, messages) => (messages.length === 0 ? '' : messages[messages.length - 1].subject),
```

Comment:
Moot because of the other comment, but note that `narrow` isn't being used here. So that and the `(state, narrow) => narrow` can be dropped.